cosmos/precompile/bank/bank.go

```diff
@@ -202,7 +202,7 @@ func (c *Contract) GetSendEnabled(
 	return res.SendEnabled[0].Enabled, nil
 }
 
-// Send implements `send(address,(uint256,string))` method.
+// Send implements `send(address,(uint256,string)[])` method.
```

Comment:
string[])

Reply:
what do you mean?
I think what I fixed is correct. It should express an array of `Coin`, which includes an amount and a denom.

Reply:
oh mb i'm goofy